bindings/core/src/method_handler/wallet.rs

```diff
@@ -92,26 +40,6 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
             let is_available = wallet.is_stronghold_password_available().await?;
             Response::Bool(is_available)
         }
-        WalletMethod::RecoverAccounts {
-            account_start_index,
-            account_gap_limit,
-            address_gap_limit,
-            sync_options,
-        } => {
-            let accounts = wallet
-                .recover_accounts(account_start_index, account_gap_limit, address_gap_limit, sync_options)
-                .await?;
-            let mut account_dtos = Vec::with_capacity(accounts.len());
-            for account in accounts {
-                let account = account.details().await;
-                account_dtos.push(AccountDetailsDto::from(&*account));
-            }
-            Response::Accounts(account_dtos)
-        }
-        WalletMethod::RemoveLatestAccount => {
-            wallet.remove_latest_account().await?;
-            Response::Ok
-        }
         #[cfg(feature = "stronghold")]
         WalletMethod::RestoreBackup {
```

Comment:
So, I feel that we need to better define the difference between `WalletMethod` and `WalletCommandMethod`. To my eyes, they are basically the same right now. Based on the names, thing like this `RestoreBackup` should be in `WalletCommandMethod` and things like `WalletCommandMethod::Transactions` should be here instead. This isn't necessarily something we need to change in this PR, but I think we should rework this (or just combine these methods because realistically we do not need both).

Reply:
I agree, I have added an entry in https://github.com/iotaledger/iota-sdk/issues/1521 to deal with it later.

Reply:
while you were away we discussed this and agree ... but not in this PR. Can adapt https://github.com/iotaledger/iota-sdk/issues/1521 if necessary?

Reply:
Yea had the same comment, suggesting manager and wallet maybe